openlr/candidate_points_getter.hpp

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include "openlr/graph.hpp"
+#include "openlr/stats.hpp"
+
+#include "indexer/index.hpp"
+
+#include "geometry/point2d.hpp"
+
+#include <cstddef>
+#include <vector>
+
+namespace openlr
+{
+class CandidatePointsGetter
+{
+public:
+  CandidatePointsGetter(size_t const maxJunctionCandidates, size_t const maxProjectionCandidates,
+                        Index const & index, Graph & graph, NStats & stat)
```

Comment:
s/NStats/stats/
?

Reply:
Not now. I have both versions of Stat class. The old one and the one I use in a new algo.

Reply:
Why not to move this to the directory v2 and namespace v2?